README.md

```diff
@@ -33,10 +33,11 @@ For a larger example and features demo:
 # Uno Platform Features
 * Supported platforms:
     * Windows 10 and Windows 11
-    * Windows 7 (via Skia+WPF)
+    * Windows 7 (via Skia Desktop)
     * iOS, Mac Catalyst and Android (via [Xamarin](https://www.visualstudio.com/xamarin/) and [.NET](https://dotnet.microsoft.com/))
```

Comment:
nitpick but Mac Catalyst is not really a platform
maybe it should be macOS and mention there's two ways to target the platform ?

Reply:
Indeed, this is outdated.